pkg/cmd/pr/merge/merge.go

```diff
@@ -293,6 +322,11 @@ func (m *mergeContext) merge() error {
 		return err
 	}
 
+	if m.addToMergeQueue() {
+		_ = m.infof("Pull request #%d will be added to the merge queue for %s when ready\n", m.pr.Number, m.pr.BaseRefName)
```

Comment:
For consistency with info lines in other cases, should this line be prefixed with an icon?
```suggestion
		_ = m.infof("%s Pull request #%d will be added to the merge queue for %s when ready\n", m.cs.SuccessIconWithColor(m.cs.Green), m.pr.Number, m.pr.BaseRefName)
```